### DISCUSSION TITLE
How to export CashValue_SE?

### DISCUSSION THREAD

**Discussion by bakerwy** [fumitoh/modelx#96] [Q&A] [answered]
Hello, 

I have a question about exporting model to improve speed.

I have tried to export model using the code **mx.read_model('C:\path..\CashValue_SE').export('C:\path..\CaseValue_SE_test')**
and it failed to export with message 'cannot add spec'

In the details message, it shows.

...mxexplorer.py line 671 in is_modelist_updated
 
 if cur['name'] == oth['name'] and cur['id'] == oth['id']:
TypeError: 'NoneType'  object is not subcriptable

Did I miss something? In the model, I did some simple modifications.
If someone gives me advises, I'd very appreciate

**Comment by fumitoh** [comment]
The detail message shows an error in spyder-modelx. Can you execute the code directly in IPython from a command prompt?

```
> ipython

In [1]: mx.read_model(r'C:\path..\CashValue_SE').export(r'C:\path..\CaseValue_SE_test')

```

**Reply by bakerwy** [reply]
I tried again and it says 'Model CashValue_SE does not have 'export''. Some model doesn't have the feature?

**Reply by fumitoh** [reply]
What does the entire error message say?

**Reply by fumitoh** [reply]
If you're using WinPython downloaded from https://lifelib.io/download.html, you should update lifelib and modelx from WinPython Command Prompt.exe:

```
> pip install -U lifelib

> pip install -U modelx

> pip install -U spyder-modelx
```

**Reply by bakerwy** [reply, marked as answer]
Finally, I created exported files!

What I have done is 
1. Deleted not related variables for the test 
2. Moved the tables under the model directory

**Reply by fumitoh** [reply]
That's great to hear. It's possible that the need for manual tweaks is due to a bug in modelx. If you can replicate the issue with a sample model and upload it, further investigation will be possible.
